index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
```

Comment:
The needs come from howe we configure gh-pages. You can configure to use the things in docs/ or in /. If you use the first, you can't access anything in /.